lambda/custom/handlers/General/LaunchRequestHandler.js

```diff
@@ -27,6 +27,7 @@ const LaunchRequestHandler = {
 					.speak(speechText)
 					.reprompt(speechText)
```

Comment:
Remove reprompt since the session has to end here because the user hasn't account linked.